sky/shell/android/org/domokit/sky/shell/SkyApplication.java

```diff
@@ -34,10 +34,11 @@
 public class SkyApplication extends BaseChromiumApplication {
     static final String SNAPSHOT = "snapshot_blob.bin";
     static final String APP_BUNDLE = "app.skyx";
+    static final String MANIFEST = "sky.yaml";
 
     private static final String TAG = "SkyApplication";
     private static final String PRIVATE_DATA_DIRECTORY_SUFFIX = "sky_shell";
-    private static final String[] SKY_RESOURCES = {"icudtl.dat", SNAPSHOT, APP_BUNDLE};
+    private static final String[] SKY_RESOURCES = {"icudtl.dat", SNAPSHOT, APP_BUNDLE, MANIFEST};
```

Comment:
My plan is to put the version and update URL in the manifest. For that, I want it to be outside the skyx file so I can do version checking without having to unpack the skyx.